### ISSUE TITLE
Use of undeclared identifier '_mediaCapturePermissionGrantType'

### ISSUE THREAD

**Issue by ethanneff** [react-native-webview/react-native-webview#2262]
Error:

<img width="2040" alt="Screen Shot 2021-11-29 at 11 27 43 AM" src="https://user-images.githubusercontent.com/2933593/143930195-ac87650d-a2e5-4295-ba8d-ad61b7d6d6c7.png">

Updating to version 11.15.0 broke iOS.

<img width="425" alt="Screen Shot 2021-11-29 at 11 26 42 AM" src="https://user-images.githubusercontent.com/2933593/143930086-3686bf41-ccc5-4990-a132-ba02e1ec7588.png">

Downgrading to 11.14.3 fixes the issue.

**Comment by Off2Race** [comment]
I had the same problem

**Comment by fnando** [comment]
Same issue, but only happens when generating the production build. Simply running `react-native run-ios` with the simulator doesn't fail with this error.

**Comment by anishmjn3** [comment]
> Error:
> 
> <img alt="Screen Shot 2021-11-29 at 11 27 43 AM" width="2000" src="https://user-images.githubusercontent.com/2933593/143930195-ac87650d-a2e5-4295-ba8d-ad61b7d6d6c7.png">
> 
> Updating to version 11.15.0 broke iOS.
> 
> <img alt="Screen Shot 2021-11-29 at 11 26 42 AM" width="425" src="https://user-images.githubusercontent.com/2933593/143930086-3686bf41-ccc5-4990-a132-ba02e1ec7588.png">
> 
> Downgrading to 11.14.3 fixes the issue.

Removing that line worked for me

**Comment by colafati** [comment]
Same issue here.

**Comment by shayananwar** [comment]
same issue here

**Comment by faradorables** [comment]
same issue here, need solution asap

**Comment by miladnikad** [comment]
+1

**Comment by rudi-one** [comment]
same issue here

**Comment by faradorables** [comment]
updated to xcode 13 and solved

**Comment by clementizard** [comment]
I found that downgrading to 11.2.3 also fixes it

**Comment by Ajmal0197** [comment]
My xcode verison: Version 12.5.1 (12E507)

Version that fixed issue:

npm i react-native-webview@11.2.3

**Comment by rudi-one** [comment]
update  to xcode 13 solving issue

**Comment by DavidAPears** [comment]
Same issue. I have...

```
- Xcode 12.5.1
- "react-native-webview": "^11.3.2"
```

Then updated to:

```
- Xcode 12.5.1
-    "react-native-webview": "^11.2.3" 
```

As per @Ajmal0197's suggestion, didn't fix it. 

Neither did:

```
- Xcode 12.5.1
- "react-native-webview": "^11.14.3"
```
:/

**Comment by NilsBaumgartner1994** [comment]
Same issue herem
will try to upgrade to xcode, but still...

**Comment by KonstantinZhukovskij** [comment]
Same issue too

**Comment by analumara** [comment]
"react-native-webview": "11.14.3"
and Xcode 13 

works fine here 🙌

**Comment by luck2011** [comment]
"react-native-webview": "11.14.3"
and Xcode 12.4.3
macOS 10.15.7

**Comment by shivenj** [comment]
**update package issue fixed**

**Comment by shivenj** [comment]
**Add Below lines in xcode , issue fixed** 

#if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 150000 /* iOS 15 */
    _mediaCapturePermissionGrantType = RNCWebViewPermissionGrantType_Prompt;
#endif

**Comment by gdoudeng** [comment]
+1

**Comment by Ajinkya-ulis** [comment]
from where may I downgrade or upgrade

**Comment by tido64** [comment]
Hi folks, this should now be resolved by #2313.

**Comment by friederbluemle** [comment]
I also just ran into this issue with Xcode 12 and react-native-webview 11.15.0
As mentioned by the issue author, downgrading to `11.14.3` helped. A better solution (which is now available) would be to **upgrade** to `11.15.1` or later, which also fixed it for me.
